default.nix

```diff
@@ -0,0 +1,24 @@
+# SPDX-FileCopyrightText: 2019 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+{ timestamp ? "19700101", docker-binaries ? null }:
```

Comment:
Are we still using timestamps btw?

Reply:
yes, autorelease has `Automatic release on <bla-bla>` default header, but this can be changed

Reply:
I suppose it doesn't really matter what we write there. We can write just "Automatic release". Normally there should be only one automatic release anyway. I think we were including the date there because it was easily available, but if we now propagate it through some scripts just to put into the release tittle I think it's an extra complication.

That being said, it's a bit out of scope of the current issue, so I don't mind if we postpone it.

Reply:
I've decided to remove timestamp usage completely